README.rst

```diff
@@ -10,7 +10,9 @@ To build::
  git submodule init
  git submodule update
  cd build-arm-mingw32ce
- ./build-mingw32ce.sh --prefix=/where/to/install
+ mkdir build-mingw32ce
+ cd build-mingw32ce
+ sh ../build-mingw32ce.sh --prefix=/where/to/install
```

Comment:
What's the purpose of this change? I don't understand.

Reply:
Calling `./build-mingw32ce.sh` from `build-arm-mingw32ce` directly caused configuring problems for binutil (i.e. the problem with "syslex.c").

Accordingly I tried to follow the steps in the build script `build-mingw32ce.sh` and found the case where for `.svn` is checked and it is suggested to create a build directory and call the build scripts from another build directory by doing `../build-mingw32ce.sh`. Although I now have a build directory inside the build directory, all configuration went right.

Perhaps it's not the optimal solution and I've been to fast with these changes or I should have opened an issue.

I have some other problems with windows header anyway and hence need some put some further effort into this... 
So maybe let's leave it like it is for now until I have the other problems fixed and reworked everything. At least some communication went on.